CHANGELOG.md

```diff
@@ -4,6 +4,32 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 0.10.1 (2018-07-12)
+
+**High-level changes**
+* Fixed memory management that was leaking memory for long-lived connections.
+* Fixed handling of dropping triggers that would lead orphan references in pg_depend.
```

Comment:
either "lead to orphaned" or simply "Fixed handling of dropping triggers on hypertables"